r1cs/gadgets/std/src/groups/mod.rs

```diff
@@ -120,10 +141,8 @@ pub trait GroupGadget<G: Group, ConstraintF: Field>:
         Ok(())
     }
 
-    /// Fixed base exponentiation, slighlty different interface from
-    /// `precomputed_base_scalar_mul`. Inputs must be specified in
-    /// *little-endian* form. If the addition law is incomplete for
-    /// the identity element, `result` must not be the identity element.
+    // what is this good for? does not use any pre-computations, just ordinary
```

Comment:
I wanted to add a slightly different interface, therefore I needed all these stuff. In general, let's keep this comments about improvements and re-styling personal, they don't apply to the purpose of this PR.